client/src/Movies/MovieList.js

```diff
@@ -35,7 +36,7 @@ function MovieDetails({ movie }) {
   const { title, director, metascore, stars } = movie;
   return (
     <div className="movie-card">
-      <h2>{title}</h2>
+      <Link to={`/movies/${movie.id}`}><h2>{title}</h2>
```

Comment:
It's preferred to separate out components/tags onto different lines for readability:
```javascript
<Link to={`/movies/${movie.id}`}>
  <h2>{title}</h2>
```